### PR TITLE
Add unit tests for Accumulo, JDBC join pushdown and BigQuery session properties

### PR DESCRIPTION
**Pull request by ebyhr**
## Description

Add helper class for testing session properties

## Documentation

(x) No documentation is needed.

## Release notes

(x) No release notes entries required.

### CODE REVIEW

**Comment by findepi** [comment]
changed PR title. you may want to adjust commit message as well

File: testing/trino-testing/src/main/java/io/trino/testing/SessionPropertiesUtil.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import static com.google.common.collect.ImmutableSet.toImmutableSet;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.testng.Assert.assertNotEquals;
+
+public final class SessionPropertiesUtil
+{
+    private SessionPropertiesUtil() {}
+
+    public static void assertDefaultProperties(SessionPropertiesProvider provider, Map<String, Object> properties)
```
[the remark this answers is not in the document]
Comment (ebyhr) [reply testing/trino-testing/src/main/java/io/trino/testing/SessionPropertiesUtil.java]: Changed to `List<PropertyMetadata>` and added `Accumulo` test.

**Comment by ebyhr** [comment]
CI hit #6315

**Comment by ebyhr** [comment]
I will reopen if necessary.

**Pull request closed without merging**